deploy/role.yaml

```diff
@@ -74,6 +74,46 @@ rules:
 - apiGroups:
     - rbac.authorization.k8s.io
   resources:
-    - clusterrolebindings
+    - rolebindings
   verbs:
-    - '*'
\ No newline at end of file
+    - '*'
+- apiGroups:
+    - mobile.k8s.io
+  resources:
+    - mobileclients
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
```

Comment:
to give push and metrics related permissions to users, the operator itself needs to have permissions on those.